src/AzureAuth/Commands/CommandAzureAuth.cs

```diff
@@ -1,34 +1,32 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Microsoft.Authentication.AzureAuth.Commands
-{
-    using System.IO.Abstractions;

```

Comment:
Using should be inside namespace in our codebase, is it?

Reply:
Yeah good catch.